AGENTS.md

```diff
@@ -382,6 +395,7 @@ describe('ComponentName', () => {
 - `/CONTRIBUTING.md` - Points to community docs
 - `/upcoming-release-notes/` - Release notes for next version
 - `/CODEOWNERS` - Code ownership definitions
+- `/packages/docs/` - Documentation website (Docusaurus), now part of the monorepo
```

Comment:
Probably don't need the "now part of" bit, it'll only be relevant for a small amount of time and there's a note present on the archived docs repo README

```suggestion
- `/packages/docs/` - Documentation website (Docusaurus)
```